packages/client/test/net/protocol/rlpxsender.spec.ts

```diff
@@ -28,7 +29,7 @@ tape('[RlpxSender]', (t) => {
 
   t.test('should receive status', (t) => {
     const rlpxProtocol = new EventEmitter()
-    const sender = new RlpxSender(rlpxProtocol)
+    const sender = new RlpxSender(rlpxProtocol as Devp2pETH)
```

Comment:
another option here could be to do
`const rlpxProtocol = new EventEmitter() as any`
or
`const rlpxProtocol = new EventEmitter() as Devp2pETH `